### ISSUE TITLE
Can't upload large files

### ISSUE THREAD

**Issue by pakipaki2000** [kzahel/web-server-chrome#170]
Hi,

Can't upload large files, it keeps crashing (to the point I have to reboot my computer) as it doesn't chunk it or anything as far as I see in Handlers.js. Anyway to modify it to store in a chunked way and avoid crash?

Thanks for this great project! This last feature and it will be just amazing

**Comment by kzahel** [comment]
Yep it reads it all into memory. This would be a great improvement (as well as a progress meter)
